tests/ami/scripts/fresh_install_rpm.sh.j2

```diff
@@ -28,19 +28,20 @@ cat /etc/*-release
 sudo yum install -y wget
 sudo yum install -y "{{python_package}}"
 
-{% if not package_to_version_is_url -%}
+# Install provided version
+{% if package_to_version_source_type == "install_script" -%}
 wget -O "install-scalyr-agent-2.sh" -q "{{installer_script_url}}"
 sudo bash ./install-scalyr-agent-2.sh --set-api-key "{{scalyr_api_key}}"
-{% endif -%}
+{% elif package_to_version_source_type in ("file", "url") -%}
+    {% if package_to_version_source_type == "url" -%}
+    wget -O to_package.rpm "{{ package_to_version }}"
```

Comment:
Probably good to be consistent with fresh install deb script and just use ``package`` instead of ``to_package`` file name here.

Reply:
I changed naming from `to/from_package` to `install_package`  and `upgrade_package`.  I hope now it easier to understand.